__tests__/index.js

```diff
@@ -243,7 +243,7 @@ if (process.platform !== 'win32') {
 }
 
 test.concurrent('should run bin command', async () => {
-  const stdout = await execCommand('bin', [], '', false);
+  const stdout = await execCommand('bin', [], '', true);
```

Comment:
If not run in a temp directory, the new behavior makes us go to the repository root. Other tests don't seem to break.